packages/client/hmi-client/src/workflow/operator/tera-operator-image-carousel.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+	<Galleria v-if="!isEmpty(images)" :value="images" :showThumbnails="false" :showIndicators="true">
+		<template #item="slotProps">
+			<img :src="slotProps.item" alt="Alt" />
+		</template>
+	</Galleria>
+	<div v-else class="no-image">
+		<i class="pi pi-image" />
+	</div>
+</template>
+
+<script setup lang="ts">
+import { isEmpty } from 'lodash';
+import { PropType } from 'vue';
+import Galleria from 'primevue/galleria';
+
+defineProps({
+	images: {
+		type: Array as PropType<string[]>,
+		default: () => []
+	}
+});
+</script>
+
+<style scoped>
+.p-galleria:deep(.p-galleria-indicators .p-galleria-indicator button) {
+	width: 0.5rem;
+	height: 0.5rem;
+}
+
+.p-galleria:deep(.p-galleria-item-container),
+.no-image {
+	height: 240px;
+	max-height: 240px;
```

Comment:
Check with @ngraham76 for the dimensions, because it should not be a square but more landscapy.

Reply:
@YohannParis @shawnyama I made a mistake in the specs. I wrote: "Preview height is not fixed but (ideally) should not exceed 240px". That should read "...should not exceed **120px**" Sorry for the confusion, I'll update the image in the feature.